tools/Crank/run-benchmarks.ps1

```diff
@@ -53,15 +56,32 @@ $crankConfigPath = Join-Path `
                     -Path (Split-Path $PSCommandPath -Parent) `
                     -ChildPath 'benchmarks.yml'
 
+$isLinuxApp = $CrankAgentVm -match '\blinux\b'
+
+$functionAppRootPath = $isLinuxApp ? "/home/$UserName/FunctionApps" : 'C:\FunctionApps';
```

Comment:
Can you change this to an if/else so PS 7 isn't required? Most people won't have that installed

Reply:
Done.